src/status_im/constants.cljs

```diff
@@ -124,7 +124,9 @@
    :currency :usd
    :appearance 0
    :log-level config/log-level
-   :webview-allow-permission-requests? false})
+   :webview-allow-permission-requests? false
+   :link-previews-enabled-sites #{}
+   :can-ask-to-preview-links true})
```

Comment:
why is this option needed?

Reply:
will be renamed but it is set to `false` when user says "Don't ask me again about link preview"